integration/cluster_test.go

```diff
@@ -449,7 +449,17 @@ func clusterMustProgress(t *testing.T, membs []*member) {
 	kapi := client.NewKeysAPI(cc)
 	ctx, cancel := context.WithTimeout(context.Background(), requestTimeout)
```

Comment:
this needs to be inside the for loop or the retry will end up using an expired context (also `cancel()` after making the `Create` call)